service/src/componentTest/resources/com/integreety/activityengine/step/find-activity.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Find activity
+
+  Scenario: find an activity by id
+  Given an existing activity with lessonId ABC
+  When the activity is requested by the returned id
+  Then the activity is returned with lessonId ABC
```

Comment:
Very minor but thinking this is slightly easier for me to read: `Then the activity with lessonId ABC is returned`